File: ISP63_BackEnd2024/angular/src/app/modelo/turnos/TurnoDTO.ts
export interface TurnoDTO {
    id: number;  // Usamos `number` para Long en TypeScript
    turno: string;
    periodoInscripcionDesde: string;  // LocalDate se puede manejar como string (formato ISO 8601)
    periodoInscripcionHasta: string;
    periodoExamenDesde: string;
    periodoExamenHasta: string;
    vigente: boolean;
}

File: ISP63_BackEnd2024/src/main/java/com/backend/ISP63/dto/TurnoDTO.java
package com.backend.ISP63.dto;

import jakarta.persistence.Column;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDate;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class TurnoDTO {

    private Long id;
    private String turno;
    private LocalDate periodoInscripcionDesde;
    private LocalDate periodoInscripcionHasta;
    private LocalDate periodoExamenDesde;
    private LocalDate periodoExamenHasta;
    private Boolean vigente;

}
